components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -3288,7 +3288,7 @@ subroutine ocn_compute_land_ice_flux_input_fields(layerThickness, normalVelocity
                blSaltScratch(nCellsAll))
       !$acc enter data create(blTempScratch, blSaltScratch)
 
-      ! Compute top drag
+      ! Compute top drag everywhere there is grounded or floating land ice
```

Comment:
@xylar See lines below where I have left the variable `landIceFraction` as the scaling factor for top drag

Reply:
Looks good!